src/components/Main/DiscordInvite/DiscordInvite.tsx

```diff
@@ -1,14 +1,19 @@
 import React from 'react';
-
+import './discordstyle.css';
 // TODO: Add text and link
 // TODO: Add graphic at the end of the page
 
 function DiscordInvite() {
   return (
     <div>
-      <h4>Want to join Girls Who Code at UVA?</h4>
-      <a href="#" target="_blank"><h2>Join our Discord server (arrow)</h2></a>
-      <img src="#" alt="Orange wave saying 'Get Started'" />
+      <div className="row">
+
+        <div className="col-lg-12 col-md-12 col-sm-12">
```

Comment:
also here it'd be `<Col sm={12}>`

if you say col-sm-12, it's 12 for all sizes small and up so the medium and large classes aren't necessary